.github/workflows/codeql-analysis.yml

```diff
@@ -12,8 +12,15 @@ on:
   schedule:
     - cron: "0 23 * * 0"
 
+permissions:
+  contents: read
```

Comment:
This one is redundant with the inner permission

Reply:
@nschonni,

You are right, it is redundant. I left it there in case any new jobs were added, that way they would start with the minimum permissions.